src/main/steps/applicant/task-list/content.test.ts

```diff
@@ -5,35 +5,63 @@ import { CommonContent } from '../../common/common.content';
 
 import { generateContent } from './content';
 const enContent = {
-  title: 'Respond to the application',
+  title: 'DA Applicant',
   statuses: {
     [SectionStatus.COMPLETED]: 'Completed',
     [SectionStatus.IN_PROGRESS]: 'In Progress',
-    [SectionStatus.NOT_STARTED]: 'Not Started',
+    [SectionStatus.TO_DO]: 'TO DO',
+    [SectionStatus.DOWNLOAD]: 'DOWNLOAD',
   },
   sectionTitles: {
-    applicantYourDetails: 'About you',
-    applicationDetails: 'Application detail',
+    aboutYou: 'About you',
+    yourApplication: 'Your application',
+    courtHearings: 'Your court hearings',
+    ordersFromCourt: 'Orders from the court',
+    yourDocuments: 'Your documents',
+    finalDecision: 'Final decision from the court',
```

Comment:
This section title doesn't match with the given prototype. As per prototype, title should be 'Orders from the court
'